test/fixture/artifact.py

```diff
@@ -50,14 +55,19 @@ def all_others(type: ArtifactType) -> list[ArtifactType]:
         return [t for t in ArtifactType if t != type]
 
 
-def _make_body(type: ArtifactType) -> NegotiationCardModel:
+def _make_body(
+    type: ArtifactType, id: str
+) -> Union[NegotiationCardModel, ValueModel]:
     """
     Make the body of the artifact for a given type.
     :param type: The artifact type
+    :param id: The identifier for the artifact
     :return: The artifact body model
     """
     if type == ArtifactType.NEGOTIATION_CARD:
         return _make_negotiation_card()
+    if type == ArtifactType.VALUE:
+        return _make_value(id)
```

Comment:
I guess this is another step now for adding new artifacts, adding it to this list and creating the fixture? Or rather, this replaces having to create the unit tests in part, to use the parameterized ones for common tests.

Reply:
That is true, I will add this to the list of items on the remaining artifact protocol port issues.